src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx

```diff
@@ -97,6 +97,24 @@ export class FileBrowserDialogComponent extends React.Component {
         return frame;
     };
 
+    private loadComplexImage = async (filename: string, expression: string) => {
+        this.imageArithmeticString = `${expression}("${filename}")`;
```

Comment:
We need `@action` when changing the observable.
I suggest we instead define a new variable here ``const imageArithmeticString = `${expression}("${filename}")`;`` and remove `this.clearArithmeticString(); this.setEnableImageArithmetic(false);` because we don't need to change the status of the UI.